.changeset/chilled-ads-learn/changes.md

```diff
@@ -1 +0,0 @@
-Add trustProxies config option to Keystone object
```

Comment:
I've reverted the changes made in the `chilled-ads-learn` PR as this solution supersedes them. I've deleted the changeset as we don't want this in the changelog, it's as if the `chilled-ads-learn` never happened :magic-hands: